scalalib/src/mill/javalib/android/AndroidAppModule.scala

```diff
@@ -677,7 +912,65 @@ trait AndroidAppModule extends JavaModule {
     emulator
   }
 
-  private val parent: AndroidAppModule = this
+  private def androidDebugKeystore: T[PathRef] = Task(persistent = true) {
+    // TODO store this key outside of the build file structure, because once build folders are deleted, new key will
+    //  be created, so app will have another signature leading to the need to remove app from the device/emulator
+    //  before installing a new build, which is annoying.
+    val debugKeystoreFile = T.dest / "debugKeyStore.jks"
```

Comment:
I wonder if we should have similar behaviour with gradle/android studio and store this in ~/.android

Reply:
I was also playing with instant APK feature, it could solve this problem